packages/wonder-blocks-link/src/components/link-core.tsx

```diff
@@ -80,7 +80,10 @@ export default class LinkCore extends React.Component<Props> {
             ...restProps,
         } as const;
 
-        // Default external icon
+        const linkUrl = new URL(href, window.location.origin);
+
+        const externalLink = linkUrl.origin !== window.location.origin;
```

Comment:
**nit (non-blocking):** It's probably a good practice to indicate that this is the evaluation of an expression to read it more as a boolean. In this case something like `isExternalLink` could help to improve readability a bit. Of course, No strong opinion on this, just a minor suggestion.